### PR TITLE
[CP-1665] Adding an information about device colour to USB descriptor

### PR DESCRIPTION
**Pull request by lukasmas**
Getting information about device colour and changing it to number. 
Pass information about device colour to usb_stack. 
Set a bcdDevice with the corresponding value.

**Your checklist for this pull request**

Make sure that this PR:
- [ ] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [ ] Has changelog entry added

### CODE REVIEW

File: module-services/service-desktop/include/service-desktop/DeviceColour.hpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <string>
+
+namespace device_colour
+{
+    uint16_t getColourVersion(const std::string &colour);
```
Comment (Maciej-Mudita) [inline module-services/service-desktop/include/service-desktop/DeviceColour.hpp:10]:
```suggestion
    std::uint16_t getColourVersion(const std::string &colour);
```

**Review by Maciej-Mudita — APPROVED** [review]

**Comment by Maciej-Mudita** [comment]
Does the documentation need to be updated?
https://appnroll.atlassian.net/wiki/spaces/CP/pages/1568374789/Endpoints

**Review by Lefucjusz — APPROVED** [review]
I'm not sure why you decided to use dynamic memory allocation for `usb_device_get_device_descriptor_struct_t` (`USB_DeviceSetBcdVersion` in `usb_device_descriptor.c`), wouldn't it be enough to just use "normal", stack-allocated variable? That PR has already been merged, so I'm asking here :P

**Comment by lukasmas** [comment]
> Does the documentation need to be updated? https://appnroll.atlassian.net/wiki/spaces/CP/pages/1568374789/Endpoints

We have decided to stop using confluence and we are going to use only the documentation in the repo

**Comment by lukasmas** [comment]
> I'm not sure why you decided to use dynamic memory allocation for `usb_device_get_device_descriptor_struct_t` (`USB_DeviceSetBcdVersion` in `usb_device_descriptor.c`), wouldn't it be enough to just use "normal", stack-allocated variable? That PR has already been merged, so I'm asking here :P

Hmm..., I don't know, I will check it :D

**Pull request merged**